internal/cli/atlas/privateendpoints/create.go

```diff
@@ -28,7 +28,7 @@ import (
 type CreateOpts struct {
 	cli.GlobalOpts
 	cli.OutputOpts
-	store    store.PrivateEndpointCreator
+	store    store.PrivateEndpointCreatorDeprecated
```

Comment:
[q] Probably just missing context, why using the deprecated creator here but the non-deprecated version for aws?

Reply:
Hi Colm! so we are using deprecated because this command is using the deprecated endpoint https://docs.atlas.mongodb.com/reference/api/private-endpoint-create-one-private-endpoint-connection. On the other hand, aws/azure are using the new endpoints https://docs.atlas.mongodb.com/reference/api/private-endpoints-service-create-one. We are discussing if keeping the commands that are using the deprecated endpoints in the spec https://docs.google.com/document/d/1zvPM8_iALD8JvBIKC-wXIjJicD0ZZxfUOqP038mN8oY/edit#